src/OrbitGl/TimerTrack.cpp

```diff
@@ -337,9 +337,7 @@ void TimerTrack::OnTimer(const TimerInfo& timer_info) {
 }
 
 float TimerTrack::GetHeight() const {
-  bool is_collapsed = collapse_toggle_->IsCollapsed();
-  uint32_t collapsed_depth = (GetNumTimers() == 0) ? 0 : 1;
```

Comment:
I will still have this variable ("collapsed_depth" with this value: std::min<uint32_t>(1, GetDepth())), for legibility

Reply:
Done.